### PR TITLE
Update styling.html.erb.md

### PR DESCRIPTION
**Pull request by nitedog**

### CODE REVIEW

File: source/carousels/styling.html.erb.md [hunk 1]
```diff
@@ -11,41 +11,21 @@ wcag_success_criteria:
 
 ## Button Size
 
-Use appropriately sized interactive elements (buttons, links) to allow [people with physical impairments](https://www.w3.org/WAI/intro/people-use-web/diversity#physical) to use the carousel more easily. This will also benefit users on mobile devices.
+Use appropriately sized buttons and links to allow [people with physical impairments](https://www.w3.org/WAI/intro/people-use-web/diversity#physical) to use the carousel more easily. This will also benefit people using touch screens.
```
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: Is it really important to have the link to "physical disabilities"? Are we doing this consistently across all tutorials?

File: source/carousels/styling.html.erb.md [hunk 2]
```diff
@@ -11,41 +11,21 @@ wcag_success_criteria:
...
 
 As a rule of thumb, {::comment} See what I did there? {:/} a minimal physical size of 9 × 9 mm for interactive elements [is recommended](https://www.w3.org/TR/mobile-accessibility-mapping/#h-touch-target-size-and-spacing) and it helps if the element is surrounded by a small amount of inactive space. While you could use millimeters in CSS, the physical rendering size might differ from device to device depending on resolution and display size. Use a minimum of 45 × 45 px in CSS to cover a 9 × 9 mm space on many devices.
```
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: Can we really say this is a rule of thumb? This is a point of discussion for WCAG 2.1 and not yet resolved. I suggest something even more tentative and a side note to say that this is being currently discussed in the context of WCAG 2.1 evelopment.

File: source/carousels/styling.html.erb.md [hunk 3]
```diff
@@ -11,41 +11,21 @@ wcag_success_criteria:
...
-More information on contrasts and other aspects of how design influences web accessibility will be provided in a future _Design Tutorial_. In the meantime find more information in the related documents below.
-
-{::nomarkdown}
-<%= ref :end %>
-{:/}
-
-Depending on the visual style you use for previous, next and play/pause buttons, you might need to similarly add a background. Another approach is to move the button outside of the area where the image is displayed.
-
-## Change shape to indicate status
+Typically space on the carousel is too limited to provide the full title of each item on the buttons. In this case the items are numbered and used at the button text. @@ In addition, hidden labels can be used to provide the full titles, especially for the next and previous buttons.
```
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: Do we use hidden labels? The earlier page seemed to indicate this.

File: source/carousels/styling.html.erb.md [hunk 4]
```diff
@@ -11,41 +11,21 @@ wcag_success_criteria:
...
-<%= ref :middle %>
-{:/}
-
-More information on contrasts and other aspects of how design influences web accessibility will be provided in a future _Design Tutorial_. In the meantime find more information in the related documents below.
-
-{::nomarkdown}
-<%= ref :end %>
-{:/}
-
-Depending on the visual style you use for previous, next and play/pause buttons, you might need to similarly add a background. Another approach is to move the button outside of the area where the image is displayed.
```
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: I think this is way too wordy but also not really useful. For example the point about browser plugin really threw me off -  had to read it several times to understand that you are talking about one possible way to check for contrast.

File: source/carousels/styling.html.erb.md (hunk 1 above)
Comment (yatil) [inline source/carousels/styling.html.erb.md]: We are not yet, but I think it would be good to integrate our resources better.

File: source/carousels/styling.html.erb.md (hunk 2 above)
Comment (yatil) [inline source/carousels/styling.html.erb.md]: Yes, I think we can say this as it is in the Mobile Accessibility Mapping document. If WCAG changes the recommendation, we can take another look.

https://www.w3.org/TR/mobile-accessibility-mapping/#h-touch-target-size-and-spacing

File: source/carousels/styling.html.erb.md [hunk 5]
```diff
@@ -11,41 +11,21 @@ wcag_success_criteria:
...
 
 As a rule of thumb, {::comment} See what I did there? {:/} a minimal physical size of 9 × 9 mm for interactive elements [is recommended](https://www.w3.org/TR/mobile-accessibility-mapping/#h-touch-target-size-and-spacing) and it helps if the element is surrounded by a small amount of inactive space. While you could use millimeters in CSS, the physical rendering size might differ from device to device depending on resolution and display size. Use a minimum of 45 × 45 px in CSS to cover a 9 × 9 mm space on many devices.
 
 ## Contrast
 
-Ensure sufficient contrast between the foreground and the background of text and interactive elements. This can be a challenge when text or buttons are positioned on top of images.
+Ensure sufficient contrast between the foreground and the background of text, links, and buttons. This can be a challenge when text or buttons are positioned on top of images. You can use a (semi-transparent) background color or move the buttons outside of the image to ensure sufficient contrast.
 
-Text needs to have a 3:1 ratio if the font size is above 18.5 px (14 pt) bold or above 24 px (18 pt) normal, and a 4.5:1 ratio otherwise. This includes text on images.
```
Comment (yatil) [inline source/carousels/styling.html.erb.md]: Why did you remove this information?

File: source/carousels/styling.html.erb.md (hunk 4 above)
Comment (yatil) [inline source/carousels/styling.html.erb.md]: We can say “ensure contrast” and not point people to resources on how to do that, but I don’t think that is a good idea. I think the section is very useful and we should keep it.

File: source/carousels/styling.html.erb.md (hunk 3 above)
Comment (yatil) [inline source/carousels/styling.html.erb.md]: Not for the previous and next buttons. (Hidden) Labels are described in other parts of the tutorial, so we don’t need this here.

**Comment by yatil** [comment]
@nitedog – I think I have tied together some of your proposed changes in e9a110eb5e0040a06c1beedae0ecd1f59e2eab01 – https://w3c.github.io/wai-tutorials/carousels/styling/

Please let me know if you feel strongly about some of the changes.

File: source/carousels/styling.html.erb.md (hunk 1 above)
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: I see your point but think it is distracting and also adds a lot of maintenance burden. Ideally the redesign will help people find information more easily without adding too many inline links.

File: source/carousels/styling.html.erb.md (hunk 2 above)
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: I know it is there but it is also a First Public Working Draft with absolutely no standing in W3C maturity. I think we need to be cautious, to not interfere with ongoing (or rather upcoming) discussions in WCAG WG. It reads too much like a requirement IMO.

File: source/carousels/styling.html.erb.md (hunk 5 above)
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: You are getting off-topic into color contrast requirements. I think it is tangential to the topic of carousels. It is distracting and making the tutorial longer than necessary.

File: source/carousels/styling.html.erb.md (hunk 3 above)
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: OK. Previously the link from thew animation page here was confusing. It is clearer now.

File: source/carousels/styling.html.erb.md (hunk 4 above)
Comment (nitedog) [inline source/carousels/styling.html.erb.md]: Again, I don't think you should say more in this tutorial than the paragraph above. You can cover everything in every tutorial.

File: source/carousels/styling.html.erb.md (hunk 2 above)
Comment (yatil) [inline source/carousels/styling.html.erb.md]: What should we say instead? I just want to provide some tangible information that people can actually use. If there is no take away here, we need to remove that section.

File: source/carousels/styling.html.erb.md (hunk 5 above)
Comment (yatil) [inline source/carousels/styling.html.erb.md]: I think this is necessary information. Just saying „Make contrast OK“ is not too useful for me. People should not go hunting for the information.

File: source/carousels/styling.html.erb.md (hunk 1 above)
Comment (yatil) [inline source/carousels/styling.html.erb.md]: You’re right. Links removed. 18c090fa42bd33cb2613d62c27cb7ae69be0b9ef
